package.json

```diff
@@ -14,13 +14,12 @@
     "jsonwebtoken": "^7.3.0"
   },
   "devDependencies": {
-    "xo": "^0.18.0"
+    "eslint-plugin-import": "^2.7.0",
```

Comment:
Any idea why this is needed? I had to add it on `probot` too.

Reply:
No! I was wondering that too. It seems to be only sometimes needed. :thinking:

Reply:
@gr2m do you know?

Reply:
no you don’t needed, `"standard"` has all you need:)

Reply:
I was getting an error on probot without this (something about missing package `eslint-plugin-import`). I spent about 30 min googling it a week or so ago and couldn't figure out how to resolve it besides installing this plugin.

Reply:
I just confirmed that it happens for me on this project. I checked out this branch locally, and run:

```
$ npm install
$ npm uninstall eslint-plugin-import
$ npm test

> github-app@3.2.0 test /Users/bkeepers/projects/probot/github-app
> standard

standard: Unexpected linter output:

Error: Failed to load plugin import: Cannot find module 'eslint-plugin-import'
Referenced from: /Users/bkeepers/projects/probot/github-app/node_modules/standard/eslintrc.json
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.resolve (internal/module.js:18:19)
    at Object.load (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/plugins.js:134:29)
    at Array.forEach (native)
    at Object.loadAll (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/plugins.js:162:21)
    at load (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/config-file.js:541:21)
    at configExtends.reduceRight (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/config-file.js:424:36)
    at Array.reduceRight (native)
    at applyExtends (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/config-file.js:408:28)
    at Object.load (/Users/bkeepers/projects/probot/github-app/node_modules/eslint/lib/config/config-file.js:566:22)

If you think this is a bug in `standard`, open an issue: https://github.com/standard/standard/issues
npm ERR! Test failed.  See above for more details.
                                                                                                                                                             ```

Reply:
I was just able to reproduce it, this is very odd. I’ll ask Feross

Reply:
Yea, I have also been getting this exact same error, but only sometimes when using standard lint? And only locally, the CI still passes, so I feel like maybe it's related to dev environments somehow? But yea after researching for a bit I couldn't find any issues relevant, but it was hard since `eslint-plugin` is really common.

Reply:
I digged some more, and it looks like an npm problem: https://github.com/standard/standard/issues/949. But downgrading to npm@5.1.0 does not help, i get the same error. The problem seems to be fixed when upgradiing eslint to 4.3 (standard has v3 still listed in its dependencies). That’s how far I got so far, I’m curious why standard is not failing on other projects of mine ... 🤔 

I also had `standard` installed globally. After I deleted that, and I run `rm -rf package-lock.json node_modules && npm install`, the tests passed?! So weird

Reply:
So I'm gonna go ahead and merge the linting PRs (that needed it) with it, and eventually on the next npm version, if we can get this working, we can remove the extraneous dependency.